src/test/java/http_server/FileHandlerTest.java

```diff
@@ -1,24 +1,22 @@
 package http_server;
 
-import org.junit.Test;
-
-import static org.junit.Assert.assertEquals;
+import file_handler.FileHandler;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
 public class FileHandlerTest {
     @Test
     public void static_files_in_public_directory_are_rendered_as_a_HTML_page_with_contents_listed() {
-        FileHandler fileHandler = new FileHandler();
         String path = "./public";
 
-        assertEquals(true, fileHandler.readDirectoryContents(path).contains("Home.html"));
-        assertEquals(true, fileHandler.readDirectoryContents(path).contains("TurtleTab.txt"));
-        assertEquals(true, fileHandler.readDirectoryContents(path).contains("japan.png"));
-        assertEquals(true, fileHandler.readDirectoryContents(path).contains("water.png"));
+        assertEquals(true, FileHandler.readDirectoryContents(path).contains("Home.html"));
```

Comment:
These tests will break if you remove these files from the directory on your computer, right? Can you think of a way to test the file handler without relying on a real directory with these specific files existing on your computer?

Reply:
I'll put in a In-Memory repository!